src/components/toast/ToastView.tsx

```diff
@@ -68,8 +68,16 @@ const ToastView = ({ content, timeout, type = 'info' }: ToastViewProps) => {
   const opacityAnim = useRef(new Animated.Value(0)).current
 
   const animOut = useCallback(() => {
-    Animated.spring(opacityAnim, { ...springConfig, toValue: 0 }).start()
-    Animated.spring(translationAnim, { ...springConfig, toValue: 0 }).start()
+    Animated.spring(opacityAnim, {
+      ...springConfig,
+      toValue: 0,
+      useNativeDriver: true
+    }).start()
+    Animated.spring(translationAnim, {
+      ...springConfig,
+      toValue: 0,
+      useNativeDriver: true
```

Comment:
That's cool! Is this something we should do for all animations?

Reply:
No, it's not supported for certain things like layout animations and gestures:

https://reactnative.dev/docs/animations#caveats